src/app/sessions/components/sessions-propositions-form/sessions-propositions-form.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { Judge } from '../../../judges/models/judge.model';
+import { Room } from '../../../rooms/models/room.model';
+import { SessionPropositionQuery } from '../../models/session-proposition-query.model';
+import * as moment from 'moment'
+
+@Component({
+    selector: 'app-sessions-propositions-form',
+    templateUrl: './sessions-propositions-form.component.html',
+    styleUrls: ['./sessions-propositions-form.component.scss']
+})
+export class SessionsPropositionsFormComponent implements OnInit {
```

Comment:
Is the name appropriate here? I don't think that this component proposes any sessions. It gathers parameters for finding resources. I think "ResourceQueryParametersFormComponent" would